Bug: SKNorm failing case

Refer to the comment from https://github.com/vprusso/toqito/issues/150#issuecomment-1561876243

For replication, it appears that the `sk_norm.py` script is failing for the following case:

```
n = 2, a = 0.5

    @pytest.mark.parametrize("n, a", [(2, 0.5), (2, -0.5), (3, 0.5), (3, -0.5)])
    def test_s1_norm_werner(n, a):
        """Test S(1) norm of a Werner state."""
        rho = werner(n, a)
        # See:
        # N. Johnston.
        # Norms and Cones in the Theory of Quantum Entanglement.
        # PhD thesis (arXiv:1207.1479)
        # Proposition 5.2.10 and Table 5.1
        expected = (1 + abs(min(0, a))) / (n * (n - a))
    
        lower_bound, upper_bound = sk_operator_norm(rho, k=1)
>       np.testing.assert_equal(np.allclose(lower_bound, expected), True)
E       AssertionError: 
E       Items are not equal:
E        ACTUAL: False
E        DESIRED: True

tests/test_matrix_props/test_sk_norm.py:60: AssertionError
```

--- a/tests/test_matrix_props/test_sk_norm.py
+++ b/tests/test_matrix_props/test_sk_norm.py
@@ -57,8 +57,8 @@ def test_s1_norm_werner(n, a):
     expected = (1 + abs(min(0, a))) / (n * (n - a))
 
     lower_bound, upper_bound = sk_operator_norm(rho, k=1)
-    np.testing.assert_equal(np.allclose(lower_bound, expected), True)
-    np.testing.assert_equal(np.allclose(upper_bound, expected), True)
+    np.testing.assert_equal(np.allclose(lower_bound, expected, atol=1e-4), True)
+    np.testing.assert_equal(np.allclose(upper_bound, expected, atol=1e-4), True)
 
 
 def test_sk_norm_hermitian_not_psd():

